src/core/mcis/networking.go

```diff
@@ -0,0 +1,210 @@
+/*
+Copyright 2019 The Cloud-Barista Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package mcis is to manage multi-cloud infra service
+package mcis
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+
+	model "github.com/cloud-barista/cb-larva/poc-cb-net/pkg/cb-network/model"
+	nethelper "github.com/cloud-barista/cb-larva/poc-cb-net/pkg/network-helper"
+	"github.com/cloud-barista/cb-tumblebug/src/core/common"
+	"github.com/cloud-barista/cb-tumblebug/src/core/mcir"
+	"github.com/go-resty/resty/v2"
+)
+
+// configureCloudAdaptiveNetwork configures a cloud adaptive network to VMs in an MCIS
+func configureCloudAdaptiveNetwork(nsId string, mcisInfo TbMcisInfo) error {
+	common.CBLog.Debug("Start.........")
+
+	// Check cb-network endpoints
+	gRPCServiceEndpoint, etcdEndpoints, err := checkCBNetworkEndpoints()
```

Comment:
```suggestion
	networkServiceEndpoint, etcdEndpoints, err := checkCBNetworkEndpoints()
```
gRPCServiceEndpoint 값에는 아마
CB_NETWORK_SERVICE_ENDPOINT=`localhost:8053`
가 입력되는 것 같네요.

gRPC는 프로토콜(방법)에 대한 상세 사항이고,
현재 코드가 REST로 처리되고 있는 것으로 봐서,
굳이 방법 자체를 변수명에 포함할 필요가 없다는 생각이 듭니다. (전반적으로)

아마 etcdEndpoints 와의 구분을 위해서 특정하신 것 같기도 합니다만.. ^^

어떻게 생각하시나요?

Reply:
@seokho-son

히스토리를 꿰뚫어 보시네요 ㅎㅎ

제가 보기에도 `networkServiceEndpoint`가 더 적절한것 같습니다 ^^